test/cli.ts

```diff
@@ -1,88 +1,88 @@
-const tape = require('tape');
-const spawn = require('tape-spawn');
-const path = require('path');
-const pkg = require('../package.json');
+import tape from 'tape';
+import spawn from 'tape-spawn';
+import path from 'path';
+import pkg from '../package.json';
 
 tape('CLI', function (t) {
   t.test('--version', function (st) {
-    var spt = spawn(st, './solcjs --version');
+    const spt = spawn(st, './solc.js --version');
```

Comment:
Do we actually want to rename this one? I think it had no `.js` extension because it's meant to be used as a shell script rather than a `.js` file meant to be imported.

Also, renaming it breaks backwards-compatibility. Though I'm not sure how much we care about that in case of this particular script. What do you think @axic?

Reply:
So the reasoning was for Typescript to correctly pick the file up and allow it to be typed as expected. I will invest some time to see if we can have no extension or rename it after completion (this is fine, within the post build script), to keep the backwards compatibility?

Reply:
once its installed, the binary reference within the package.json file should allow it to be called like "solcjs --version" as expected (as before).

Reply:
ok, if it keeps the name for the final script that you're supposed to run it would be fine.

Reply:
So please correct me if im reading this wrong but i'm 99.9% sure this is how its expected.

https://docs.npmjs.com/cli/v7/configuring-npm/package-json#bin

Reply:
Oh, you're right. npm will create a `solcjs` symlink even though the file itself is called `solc.js`. Yeah, I think this renaming should not be a backwards-compatibilty problem then.